building.md

```diff
@@ -0,0 +1,31 @@
+# Building NV2
+
+The nv2 repo contains the following:
+
+- `nv2` - A CLI for signing and verifying with Notary v2
+- `docker-generate` - Extends docker with `docker generate` to create locally persisted manifest for signing, without having to push to a registry.
+- `docker-nv2` - Extends  docker with `docker nv2 notary` to enable, sign and verify Notary v2 signatures on `docker pull`
+
+This plugin requires [golang](https://golang.org/dl/) with version `>= 1.16`.
+
+## Windows with WSL
+
+- Build the binaries, installing them to:
+  - `~/bin/nv2`
+  - `~/.docker/cli-plugins/docker-generate`
+  - `~/.docker/cli-plugins/docker-nv2`
+  ```shell
+  make install
```

Comment:
We need a `git clone` and `cd` step in here before running make. Since a lot of the other steps are pulling in external projects, it might be useful to emphasize this is for the current repo.

Reply:
I've added:
```bash
git clone https://github.com/notaryproject/nv2.git
cd nv2
git checkout prototype-2
make install
```